app/brave_settings_strings.grdp

```diff
@@ -1517,6 +1517,67 @@ Leo. This action cannot be undone.
   <message name="IDS_SETTINGS_LEO_ASSISTANT_MANAGE_URL" desc="Label for managing accounts url">
     Manage premium subscription
   </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_BYOM_LABEL" desc="A generic title to use for headings">
+    Bring your own model
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_DELETE_MODEL_CONFIRMATION" desc="Confirmation message for deleting a model">
+    Are you sure you want to delete this model? If this model is being used in any active conversation then that conversation will change to use Leo's default model.
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_ADD_MODEL_LABEL" desc="A title for adding a custom model">
+    Add model
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_EDIT_MODEL_LABEL" desc="A title for adding a custom model">
+    Edit model
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_LABEL" desc="An input title for adding a custom model">
+    Label
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_REQUEST_NAME" desc="An input title for models request name. This is gets sent to the remote server.">
+    Model request name
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_SERVER_ENDPOINT" desc="An input title for server url.">
+    Server endpoint
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_API_KEY" desc="An input title for server API key.">
+    API Key
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_LABEL_TOOLTIP_INFO" desc="A brief description for the input title for adding a custom model">
+    The name of the model as it will appear in the model selection
+              menu
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_REQUEST_NAME_TOOLTIP_INFO" desc="A brief description for the input title for adding model request name">
+    The name of the model as it should appear in the request to the serving framework, i.e. phi-3 (note that if this name doesn’t match exactly what is expected by the serving framework the integration will not work)
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_SERVER_ENDPOINT_TOOLTIP_INFO" desc="A brief description for the input title for adding server endpoint url">
+    The url where your serving framework is listening for requests. If you are not sure, check the serving framework documentation. E.g. for Ollama, this is always http://localhost:11434/api/chat
```

Comment:
the final string should be `http://localhost:11434/v1/chat/completions`

Reply:
done